krummstab/commands/correct.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import pathlib
+import subprocess
+
+from .. import config, sheets
+
+
+def run_command_and_wait(command: str) -> None:
+    """
+    Executes a shell command and waits for it to finish.
+    """
+    logging.info(f"Running: {command}")
+    process = subprocess.Popen(command, shell=True)
```

Comment:
- Use `subprocess.run` and provide command as list of strings instead of single string, where the first element is the command itself and the following elements are parameters. Come up with a magic word that has to be part of the config string list that will be replaced by the actual path to the PDF file here.
- Avoid `shell=True`.
- For reference: `cmd = [arg.format(**locals()) for arg in cmd]`